indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -108,11 +114,51 @@
         $('#select-none').on('click', function() {
             $('table.i-table input.select-row').prop('checked', false).trigger('change');
         });
+
+        $('.js-dialog-send-email').ajaxDialog({
+            getExtraData: function() {
+                return {registration_id: getSelectedRows()};
+            }
+        });
+
+        $('.js-pdf-export').on('click', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            $('.registrations form').attr('action', $this.data('href')).submit();
+        });
+
+        $('.registrations').on('indico:confirmed', '.js-delete-registrations', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            var selectedRows = getSelectedRows();
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                data: {registration_id: selectedRows},
+                traditional: true,
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function() {
+                    for (var i = 0; i < selectedRows.length; i++) {
+                        var row = $('#registration-' + selectedRows[i]);
+                        row.fadeOut('fast', function() {
+                            row.remove();
```

Comment:
I'm pretty sure this will only remove the last row (`row` is bound to the success function scope, so it's the same variable in each iteration). Easiest way to avoid this is use `$.each()` to iterate over `selectedRows`.